scripts/combine_metadata.py

```diff
@@ -45,7 +44,12 @@ def parse_args():
     # READ IN METADATA FILES
     metadata = []
     for (origin, fname) in zip(args.origins, args.metadata):
-        data, columns = read_metadata(fname)
+        data = read_metadata(fname)
+        columns = ["strain"] + data.columns.tolist()
+        data = data.to_dict(orient="index")
+        for strain in data.keys():
+            data[strain]["strain"] = strain
```

Comment:
This can also be accomplished using pandas functions:

```suggestion
        data = read_metadata(fname)
        data = data.rename_axis('strain').reset_index()
        data = data.to_dict(orient="index")
```

Reply:
I tried a similar approach, too, but it doesn't produce quite the same output, since resetting the index makes the index values integers again and then orienting the dict by index produces a dict indexed by those (meaningless) integers. Also, when you read in the file, the index is already named "strain".

The other approach that worked for me was (using data from the `nextstrain-ci` profile):

```python
data = read_metadata("results/europe/metadata_adjusted.tsv.xz")
data.insert(0, "strain", data.index.values)
columns = data.columns
data = data.to_dict(orient="index")
```

I can push this instead, if it seems clearer. It is probably marginally faster, although see other comment about the general need to optimize this script. :D

Reply:
Ohh I see! That looks good.